soccer/gameplay/plays/restarts/our_free_kick.py

```diff
@@ -16,7 +16,7 @@ class OurFreeKick(standard_play.StandardPlay):
     BumpKickPower = 0.01
     FullKickPower = 1
     MaxShootingAngle = 80
-    # Untested as of now
+    # Untested as of now - I didn't test this either!!!
```

Comment:
This comment made my day :)

Reply:
on the one hand - good :), on the other- I didn't mean to include this commit... so there's that